tests/system_tests/test_artifacts/test_data_types.py

```diff
@@ -435,3 +435,26 @@ def mock_get_entry_name(incr_table, key):
     assert incremental_table.data == expected_full_data
 
     verification_run.finish()
+
+
+def test_incremental_tables_cannot_be_logged_on_multiple_runs(
+    user,
+    test_settings,
+):
+    """
+    Test that incrementally logged tables cannot be logged to multiple runs.
+    """
+    wandb.setup(wandb.Settings(reinit="create_new"))
```

Comment:
Feature isn't needed for this test; you should be able to just create two separate runs one after another.

```python
incr_table = ...

with wandb.init(mode="offline") as run:
    # log incr_table to run

with wandb.init(mode="offline") as run:
    # assert logging incr_table to the second run raises
```

I also recommend using `offline` mode and removing the `user` fixture to make the test faster.